src/Sulu/Bundle/PageBundle/Document/BasePageDocument.php

```diff
@@ -240,7 +240,7 @@ class BasePageDocument implements
      *
      * @var array
      */
-    protected $permissions;
+    protected $permissions = [];
```

Comment:
this make sure that also when no `setPermissions` is called the permissions are as defined an array and not null.